frontend/src/components/common/TooltipToggleButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { ToggleButton, Tooltip, ToggleButtonProps } from "@mui/material";
+
+const style = {
+	toggleButton: {
+		width: "25px",
+		height: "25px",
+		minWidth: "25px",
+		padding: "0",
+		margin: "2px",
+		border: "none",
+		fontWeight: "bold",
+	},
+};
+
+function TooltipToggleButton({
+	title,
+	children,
+	...props
+}: ToggleButtonProps & { title?: string }) {
+	return (
+		<Tooltip title={title} placement="top">
+			<ToggleButton sx={style.toggleButton} {...props}>
+				{children}
+			</ToggleButton>
+		</Tooltip>
+	);
+}
+
+export default TooltipToggleButton;
```

Comment:
I moved to `frontend/src/components/common`
and, refactor prop type. (by using mui)